developer-experience/recipes/bridge-to-kubernetes.md

```diff
@@ -0,0 +1,60 @@
+
+# Bridge to Kubernetes
+
+Bridge to Kubernetes allows developers to seamlessly work on services both in a shared cluster and locally in debug mode.
+
+To learn more about Bridge to Kubernetes:
+
+- [Introduction](https://code.visualstudio.com/docs/containers/bridge-to-kubernetes)
+
+- [How it works](https://docs.microsoft.com/en-us/visualstudio/containers/overview-bridge-to-kubernetes)
+
+- [Configuration](https://docs.microsoft.com/en-us/visualstudio/containers/configure-bridge-to-kubernetes)
+
+- [VS Code Setup](https://docs.microsoft.com/en-us/visualstudio/containers/bridge-to-kubernetes?view=vs-2019)
+
+- [Repository](https://github.com/Microsoft/mindaro)
+
+## Useful Scenarios
+
+We believe this technology can be most helpful...
+
+- if a developer needs to test/debug a new or changed service against an ecosystem of services.
+
+- if a developer does not have the domain knowledge to deploy all the micro services. This allows them to work against a "blackbox" set of already deployed services.
+
+We found the technology very easy to implement so it can probably be used for a wide of cases with minimal training.
```

Comment:
nit: it looks a bit weird with personal reflection here - who is we? I would be more direct "The technology can be used for a wide variety of cases with minimal training"

also in the nit "wide of cases" => wide variety of cases or many different cases

Reply:
I changed it as you suggested. This was written as part of a hackweek. So "we" was the team writing it.